Xamarin.MacDev/IPhoneDeviceType.cs

```diff
@@ -46,6 +46,7 @@ public enum AppleDeviceFamily
 		IPad   = 2,
 		TV     = 3,
 		Watch  = 4,
+		IPod   = 5,
```

Comment:
I would probably prefer that the IPod value be added with the other I-devices.

IPod,
IPhone,
IPad,
...

I haven't looked at the other PR yet, but keep in mind that this enum and IPhoneDeviceType get converted back and forth in a few places, so we need to make sure that IPod maps to IPhoneDeviceType.IPhone most likely.